service/src/main/java/uk/nhs/adaptors/oneoneone/config/DeviceProperties.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.adaptors.oneoneone.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Component
+@ConfigurationProperties
+@Getter
+@Setter
+public class DeviceProperties {
```

Comment:
root level properties are common adaptor properties, not device specific. Maybe let's name this class "OneOneOneProperites"